### PR TITLE
Node consistency 81 90

### PR DESCRIPTION
**Pull request by martinstacey**
### Purpose
The following PR addresses nodes' names and descriptions to make them more descriptive for beginners.

**81) NODE: List.Deconstruct**
-CHANGES:
Output description:
first: type: var[]..[] (first item in the list)
rest: type: var[]..[] (rest of the list)

**82) NODE: List.Map**
-CHANGES:
Input: fx -> function
Output: mapped -> list
Output description list: type: var[]..[] (mapped list)

**83) NODE: List.Cycle**
-CHANGES:
Output description list: type: var[]..[] (list of repeated lists)

**84) NODE: Sequence**
-CHANGES:
Input description: Amount of numbers or letters in the sequence
Output seq to list
Output description list: type: var[]..[] (New list)

**85) NODE: Range**
-CHANGES:
Output seq to list
Output description list: type: var[]..[] (New list)

**86) NODE: TimeSpan.Components**
-CHANGES:
Output description:
days: type: int (amount of days)
hours: type: int (amount of hours)
minutes: type: int (amount of minutes)
seconds: type: int (amount of seconds)
milliseconds: type: int (amount of milliseconds)

**87) NODE: Object.IsNull**
-CHANGES:
Description: Determines if the given object is null.

**88) NODE: Object.Identity**
-CHANGES:
Input: obj to object
Output : var[]..[] to object
Output object description: Same object

**89) NODE: DateTime.TimeOfDay**
-CHANGES:
Output: var to TimeSpan
Output Description: TimeSpan since midnight

**90) NODE: DateTime.Date**
-CHANGES:
Output: var to date
Output Description: date from DateTime

image

### Reviewers
@SHKnudsen
@StudioLE
@MarkThorley
@anrova

### FYIs
-No FYI

### CODE REVIEW

File: src/Libraries/CoreNodes/DateTime.cs [hunk 1]
```diff
@@ -157,6 +157,8 @@ public static System.DateTime FromString(string str)
         ///     Extracts only the date from a DateTime. Time components are set to 0.
         /// </summary>
         /// <param name="dateTime">A DateTime.</param>
+        /// <returns name="date">date from DateTime</returns>
+        /// <search>Date,DataTime.Date</search>
```
Comment (SHKnudsen) [inline src/Libraries/CoreNodes/DateTime.cs]: From the review on one of the PRs on Dynamo:

```I don't think the name of the node needs to be repeated in the search terms as the node name is used in search by default.```

File: src/Libraries/CoreNodes/DateTime.cs [hunk 2]
```diff
@@ -358,6 +361,11 @@ public static System.TimeSpan FromString(string str)
         ///     Extracts the individual components of a TimeSpan.
         /// </summary>
         /// <param name="timeSpan">A TimeSpan.</param>
+        /// <returns name="days">Amount of days on TimeSpan</returns>
+        /// <returns name="hours">Amount of hours on TimeSpan</returns>
+        /// <returns name="minutes">Amount of minutes on TimeSpan</returns>
+        /// <returns name="seconds">Amount of seconds on TimeSpan</returns>
+        /// <returns name="milliseconds">Amount of milliseconds on TimeSpan</returns>
```
Comment (SHKnudsen) [inline src/Libraries/CoreNodes/DateTime.cs]: Do we need the `on TimeSpan` on all of these?

File: src/Libraries/CoreNodes/DateTime.cs (hunk 1 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/DateTime.cs]: Addressed in 5c96603

File: src/Libraries/CoreNodes/DateTime.cs (hunk 2 above)
Comment (martinstacey) [reply src/Libraries/CoreNodes/DateTime.cs]: Addressed in 983b5dc

**Pull request merged**